README.md

```diff
@@ -44,16 +44,60 @@ tab: `/admin/settings/models`.
 
 ## Patching
 
-This project requires us to make changes requested by the project owner to align with the organization. The upstream
+This project requires us to make changes requested by the upstream project to align with the organization. The upstream
```

Comment:
"This project requires us to make changes requested by the _upstream project_ to align with the organization. The _upstream project_ may not appreciate some of these changes"

So "upstream" don't want their own changes? Not sure what the intended message is but this should propably be re-written.